RestaurantReservation.Db/Services/CustomerService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using RestaurantReservation.Db.Models;
+using RestaurantReservation.Db.Repositories;
+
+namespace RestaurantReservation.Db.Services
+{
+    public class CustomerService
+    {
+        private RestaurantReservationDbContext _context;
+        private CustomersRepository customersRepository;
```

Comment:
this will remine the same, u wont use ICustomerRepository here until u take the web api course and learn how to register your services using DI

Reply:
OK